Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveM.cpp

```diff
@@ -62,6 +62,7 @@ void FiniteDifferenceSolver::MacroscopicEvolveM (
         // static constexpr amrex::Real Ms = 1e4;
         // Real constexpr cons1 = -mag_gamma_interp; // should be mu0*gamma, mu0 is absorbed by B used in this case
         // Real constexpr cons2 = -cons1*alpha/Ms; // factor of the second term in scalar LLG
+        Real M_normalized_error = 0.1; // normalization error of M field for checking
 
```

Comment:
note... in a future PR let's put this in materialproperties and for reading for the inputs file (you can close this comment once you read)

Reply:
Got it!